### PR TITLE
Implement memory

### PR DESCRIPTION
**Pull request by Sammers21**
_Closes #17

### CODE REVIEW

**Comment by Sammers21** [comment]
@g4s8, review the pull request please

**Comment by 0crat** [comment]
This pull request [#22](https://github.com/artipie/helm-adapter/issues/22) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/8358d7c9-38e3-4c04-9583-a007f382324d); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/helm/PushChartSlice.java [hunk 1]
```diff
@@ -66,6 +70,26 @@ public Response response(final String line,
         return new AsyncResponse(this.response(body));
     }
 
+    /**
+     * Convert buffers into a byte array.
+     * @param bufs The list of buffers.
+     * @return The byte array.
+     */
+    static byte[] bufsToByteArr(final List<ByteBuffer> bufs) {
```
Comment (paulodamaso) [inline src/main/java/com/artipie/helm/PushChartSlice.java:78]: @Sammers21 How about using `com.artipie.asto.Concatenation` here?

Comment (Sammers21) [reply src/main/java/com/artipie/helm/PushChartSlice.java:78]: No, it is less efficient

**Comment by 0crat** [comment]
Job was finished in 2 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@paulodamaso please make sure you will find at least 3 issues during next CR, thanks

**Comment by paulodamaso** [comment]
@sereshqua OK, I will, thank you

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**